python/tidy/servo_tidy_tests/rust_tidy.rs

```diff
@@ -3,12 +3,12 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 use    app_units::Au;
-use azure::azure_hl::{ AntialiasMode, Color, ColorPattern, CompositionOp };
-use azure::azure_hl::{AntialiasMode, Color,
+use raqote::{GradientStop, Source, SolidSource};
+use raqote::{Source, SolidSource},
```

Comment:
I just picked random structs, not sure if I'm still matching the intention of the tests.

Reply:
@SimonSapin I just noticed -- I didn't preserve the extraneous whitespace on the first line here.

Reply:
Does `./mach test-tidy --self-test` succeed for you? If not adjust files until it does, make or amend commits, and push or force-push to this PR again.